docs/src/pages/guides/FHIRSearchConfiguration.md

```diff
@@ -34,11 +34,10 @@ The IBM FHIR Server supports compartment searches based on the CompartmentDefini
 ### 1.1 Tenant-specific parameters
 To configure tenant-specific search parameters, create a file called `extension-search-parameters.json`, placing it in the `${server.config.dir}/config/<tenant-id>` directory. For example, the `${server.config.dir}/config/acme/extension-search-parameters.json` file would contain the search parameters for the `acme` tenant, while `${server.config.dir}/config/qpharma/extension-search-parameters.json` would contain search parameters used by the `qpharma` tenant.
 
-When the IBM FHIR Server processes a request associated with the `acme` tenant, the server uses the built-in search parameters and the user-defined search parameters defined in the `acme` tenant's `extension-search-parameters.json` file. Likewise, when processing a request associated with the `qpharma` tenant, the server uses the built-in search parameters and the user-defined search parameters defined in the `qpharma` tenant's `extension-search-parameters.json` file.
-
-If a tenant-specific extension-search-parameters.json file does not exist, the server falls back to the global `extension-search-parameters.json` file found at `${server.config.dir}/config/default/extension-search-parameters.json`. For performance reasons, we recommend having an `extension-search-parameters.json` for each tenant.
+If a tenant-specific extension-search-parameters.json file does not exist, the server falls back to the default `extension-search-parameters.json` file found at `${server.config.dir}/config/default/extension-search-parameters.json`. For performance reasons, we recommend having an `extension-search-parameters.json` for each tenant.
```

Comment:
We don't hit it after startup? does it cause issues now?

Reply:
For better or worse, the new ExtensionSearchParametersResourceProvider impl will work the same way as the previous behavior...it will use TenantSpecificSearchParameterCache to cache the `extension-search-parameters.json` contents.  So the same guidance still applies, even though the parameters will only ever be loaded on startup.